lib/galaxy_test/driver/driver_util.py

```diff
@@ -563,7 +563,14 @@ def uvicorn_serve(app, port, host=None):
     import asyncio
     from uvicorn.server import Server
     from uvicorn.config import Config
-    config = Config(app, host=host, port=int(port))
+
+    access_log = os.environ.get('GALAXY_UVICORN_ACCESS_LOG', None)
```

Comment:
```suggestion
    access_log = os.environ.get('GALAXY_TEST_UVICORN_ACCESS_LOG', None)
```
All the variables that control Galaxy via the test driver use GALAXY_TEST_* prefixes.